concent_api/core/payments/bankster.py

```diff
@@ -456,6 +460,35 @@ def get_provider_payment_info(
     return (amount_paid, amount_pending)
 
 
+def remove_duplicated_transactions_from_deposit_claims(
+    claims_of_settlement_payments: QuerySet,
+    list_of_forced_payments: List[ForcedPaymentEvent],
+) -> QuerySet:
+    """
+    Removes duplicates from claims of settlement payments with pending transaction
+    which are on list of forced payments by adding exclusions to DepositClaim queryset.
+    """
+    for force_payment in list_of_forced_payments:
+        try:
+            matching_claim = claims_of_settlement_payments.get(tx_hash=force_payment.tx_hash)
+        except DepositClaim.DoesNotExist:
+            continue
+
+        # If the information in the database differs from the information on the blockchain,
+        # aborts the settlement and responds with HTTP 500.
+        if (
+            matching_claim.closure_time != force_payment.closure_time or
+            matching_claim.amount != force_payment.amount
+        ):
+            raise BanksterTransactionMismatchError(
```

Comment:
consider log this situation before raising `BanksterTransactionMismatchError`

Reply:
ok, unhandled exception are handled by the middleware... if so, the question is if we even need a crash logger...

Reply:
As discussed, not necessary in this case.